pyglmnet/pyglmnet.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 from scipy.special import expit
+import utils as utils
```

Comment:
you should use relative imports for importing functions, otherwise people might end up importing this: https://pypi.python.org/pypi/utils